Sources/TuistSupport/Utils/WarningController.swift

```diff
@@ -27,11 +27,11 @@ public final class WarningController: WarningControlling {
 
     public static var shared: WarningControlling = WarningController()
 
-    private init() {}
+    init() {}
```

Comment:
I made this one `internal` to be able to use it from tests targets using `@testable import`.